src/test/util_sds_check/util_sds_check.c

```diff
@@ -0,0 +1,185 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/* Copyright 2025, Hewlett Packard Enterprise Development LP */
+
+/*
+ * util_sds_check.c -- unit test for the shutdown_state_check() function
+ */
+
+#include "util_sds_check.h"
+
+/* pool_sds - SDS read from the pool header (persistent) */
+static struct shutdown_state Pool_sds;
+/*
+ * curr_sds - SDS calculated ad hoc considering all the involved PMem DIMMs
+ * their UUIDs and USCs (volatile)
+ */
+static struct shutdown_state Curr_sds;
+static struct pool_replica Rep;
+
+FUNC_MOCK(shutdown_state_reinit, void, struct shutdown_state *curr_sds,
+	struct shutdown_state *pool_sds, struct pool_replica *rep)
+FUNC_MOCK_RUN_DEFAULT {
+	ASSERTeq(curr_sds, &Curr_sds);
+	ASSERTeq(pool_sds, &Pool_sds);
+	ASSERTeq(rep, &Rep);
```

Comment:
Please replace all asserts with the variant dedicated for unit test. All occurrences in this file.

```suggestion
	UT_ASSERTeq(curr_sds, &Curr_sds);
	UT_ASSERTeq(pool_sds, &Pool_sds);
	UT_ASSERTeq(rep, &Rep);
```

Reply:
Done.